keyboards/handwired/scottokeebs/scottosplit/info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoSplit",
+    "maintainer": "joe-scotto",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        //  3, 4, 5, 6, 7, 8, 9, A3, A2, A1
+        "cols": ["D0", "D4", "C6", "D7", "E6", "B4", "B5", "F4", "F5", "F6"]
+
+        // 15, 14, 16, 10
+        "rows": ["B1", "B3", "B2", "B6"]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x5_3": {
```

Comment:
This looks like it matches the split_3x5_3 layout, actually.

It might be worth adding an alias for that, and community layout support.